### ISSUE TITLE
License question

### ISSUE THREAD

**Issue by pombredanne** [unixpackage/unixpackage.github.io#2]
Hi:
If I were to reference or include these JSON data in https://github.com/nexB/scancode-toolkit/ which is Apache-licensed what would this mean? I am a tad at lost to understand what the AGPL means for JSON data.
Thank!

**Comment by pombredanne** [comment]
Well I guess this is something you are working on per https://github.com/unixpackage/unixpackage/issues/5

**Comment by colmoconnor-loveholidays** [comment]
Yep, it's not AGPL any more it's BSD. Include as you like.

**Comment by crdoconnor** [comment]
Yep, it's not AGPL any more it's BSD. Include as you like. No restrictions, although a shout out would be nice, as would pull requests with new packages.

**Comment by pombredanne** [comment]
@crdoconnor thanks. that was quick! I will sure contribute back my mods if any.

**Comment by pombredanne** [comment]
@boweevil I assume you are OK with the license change?

**Comment by crdoconnor** [comment]
Oh yeah, he has to give the go ahead too.

**Comment by jasonphillipcarpenter** [comment]
Sorry, just saw this.  Yeah, I'm fine with the change.
